toolchain/check/handle_let.cpp

```diff
@@ -10,6 +10,10 @@ namespace Carbon::Check {
 
 auto HandleLetDecl(Context& context, Parse::NodeId parse_node) -> bool {
   auto value_id = context.node_stack().PopExpr();
+  if (context.parse_tree().node_kind(context.node_stack().PeekParseNode()) ==
+      Parse::NodeKind::ParamList) {
```

Comment:
```suggestion
  if (context.node_stack().PeekIs<Parse::NodeKind::ParamList>()) {
```
... although it would probably be more future-proof to use `PopIf<BindingPattern>` below instead, and reject all non-`BindingPattern` cases.